nengo_gui/static/nengo.js

```diff
@@ -99,5 +104,15 @@ Nengo.draw_legend = function(parent, labels, color_func){
               .html(function(d, i) {
                     return labels[i];
                });
+
+    // expand the width of the svg to the length of the longest string
+    var label_list = $("#id"+uid+" .legend-label").toArray();
+    var longest_label = label_list.sort(
+                            function (a, b) { return b.getBBox().width - a.getBBox().width; }
+                        )[0];
```

Comment:
Why sort the items and then take the first one?  Why not just find the max in the array?

(It just feels weird to do an O(N**2) operation when an O(N) would work fine)